lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1602,6 +1602,39 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => 'Controls auto-instrumentation of Sinatra at start up. May be one of: `auto`, `prepend`, `chain`, `disabled`.'
         },
+        :'instrumentation.stripe' => {
+          :default => 'enabled',
+          :public => true,
+          :type => String,
+          :allowed_from_server => false,
+          :description => 'Controls auto-instrumentation of Stripe at start up. May be one of: `enabled`, `disabled`.'
+        },
+        :'stripe.user_data.include' => {
+          default: NewRelic::EMPTY_ARRAY,
+          public: true,
+          type: Array,
+          dynamic_name: true,
+          allowed_from_server: false,
+          :transform => DefaultSource.method(:convert_to_list),
+          :description => <<~DESCRIPTION
+            An array of strings to specify which keys inside a Stripe event's `user_data` hash should be reported
+            to New Relic. Each string in this array will be turned into a regular expression via `Regexp.new` to
+            permit advanced matching. Setting the value to `"."` will report all `user_data`.
```

Comment:
> Setting the value to `"."` will report all `user_data`.

Technically - given that the value is an array of strings and not a single string - it would be `["."]`. I think you should either make that change to the backticked code and leave the rest of the sentence unchanged or rewrite it to say something like "Add `"."` to the array to match all hash keys".

Reply:
Good point! Updated to the hash option https://github.com/newrelic/newrelic-ruby-agent/pull/2180/commits/8cd8f196a25cb78da3da8b2130fb11b3d9202cb2